DuckDuckGo/NetworkProtection/AppTargets/DeveloperIDTarget/NetworkProtectionIPCTunnelController.swift

```diff
@@ -160,11 +161,11 @@ extension NetworkProtectionIPCTunnelController: TunnelController {
     private func log(_ error: Error) {
         switch error {
         case RequestError.notAuthorizedToEnableLoginItem:
-            os_log("🔴 IPC Controller not authorized to enable the login item", log: .networkProtection)
+            Logger.networkProtection.debug("IPC Controller not authorized to enable the login item")
```

Comment:
I think it could be error

Reply:
agree, log type changed and error description added